ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -42,6 +42,7 @@ set MIXER_FILE none
 set MK_MODE none
 set MKBLCTRL_ARG ""
 set OUTPUT_MODE none
+set PARAM_DEFAULTS_VER 1
```

Comment:
Probably worth breaking this out of the list and adding a comment to explain the usage of PARAM_DEFAULTS_VER and when developers should be changing it.

Reply:
Done.